docs/0.1/getting-started/quick-start.md

```diff
@@ -1,46 +1,178 @@
+
+
+## Get an Access Token
+
+You will need an access token to push data into the Microshare platform. Follow the steps listed below to get an access token.
+
+Note: you will need an account to get an access token
+
+
+1.  Navigate to the [Microshare platform](https://app.microshare.io) and sign in using your credentials
+2.  Click the "Manage" button in the top toolbar
+3.  Click the "Keys" button in the left toolbar
+4. Click on the "Create New App" button at the top right of the screen
+5. Type the app name and any appropriate notes relating to the app, then click "Create App"
+6. Click on the newly created API Key to copy it 
+7. Open Postman
+8. Make a Post call to https://auth.microshare.io
+    * See below for an example of the parameters for the Post Call:
+
+
+{% highlight JSON%}
 {
-  "hello": "world"
+    "key": "grant_type",
+    "value": "password",
+    "equals": true,
+    "description": ""
+},
+{
+    "key": "client_id",
+    "value": "12345678-ABCD-9123-EFGH-456789IJKLMN",
+    "equals": true,
+    "description": ""
+},
+{
+    "key": "scope",
+    "value": "ALL:ALL",
+    "equals": true,
+    "description": ""
+},
+{
+    "key": "username",
+    "value": "example@microhare.io",
+    "equals": true,
+    "description": ""
+},
+{
+    "key": "password",
+    "value": "samplePassword",
+    "equals": true,
+    "description": ""
+},
+{% endhighlight %}
+
+* The parameters of the Post call explained:
+    * The key "grant_type" should have a value of "password"
+    * The key "client_id" should have the API Key as it's value
+    * The key "scope" should have a value of "ALL:ALL"
+    * The key "username" should have your email as it's value
+    * The key "password" should have your accounts password as it's value
+
+* This will return a JSON with an access token. The access token will be a series of letters and numbers that is associated with the "access_token" Key. You can post data to Microshare with this access token.
+
+## Post Data to Microshare
+
+Make a post call using the same parameters as before, but with two notable additions. 
+
+1. Include the data you wish to upload into the body of the call in JSON format. For example:
+
+    {"Test":"Data"}
+    
+2. Include a header with a key of "recType" 
+    * This will create a space for the data and attribute the above value as the recType's name  
+        * It is recommended that your first recType follow a simple naming convention I.E. "yourFirstName.yourLastName"
+
+
+## Make a Rule 
+
+A rule governs the data you have uploaded in a specific recType. You can adjust who can see the data and what they can do with it. In this section we will allow a user of your choosing to read and write to the data you have posted to microshare.
+
+1. Navigate to [our platform](https://app.microshare.io)
+2. Click the "Manage" button in the top toolbar
+3. Click the "Rules" button in the left toolbar
+4. Click the the "Create" button
+5. Input the Rule name in the top most text field
```

Comment:
Hey, the numbering system is still off here. It only shows 5s for the following 7 lines. The same with lines 137 -141, they only show 4s.